Description: replace ava by tape
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-03-31

--- a/test.js
+++ b/test.js
@@ -1,7 +1,7 @@
 import process from 'node:process';
 import path from 'node:path';
 import {fileURLToPath} from 'node:url';
-import test from 'ava';
+import test from 'tape';
 import {npmRunPath, npmRunPathEnv} from './index.js';
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
@@ -16,6 +16,7 @@
 		npmRunPathEnv({env: {PATH: 'foo'}}).PATH.split(path.delimiter)[0],
 		path.join(__dirname, 'node_modules/.bin'),
 	);
+	t.end();
 });
 
 test('the `cwd` option changes the current directory', t => {
@@ -23,6 +24,7 @@
 		npmRunPath({path: '', cwd: '/dir'}).split(path.delimiter)[0],
 		path.normalize('/dir/node_modules/.bin'),
 	);
+	t.end();
 });
 
 test('the `cwd` option can be a file URL', t => {
@@ -30,11 +32,13 @@
 		npmRunPath({path: '', cwd: new URL('file:///dir')}).split(path.delimiter)[0],
 		path.normalize('/dir/node_modules/.bin'),
 	);
+	t.end();
 });
 
 test('push `execPath` later in the PATH', t => {
 	const pathEnv = npmRunPath({path: ''}).split(path.delimiter);
 	t.is(pathEnv[pathEnv.length - 2], path.dirname(process.execPath));
+	t.end();
 });
 
 test('can change `execPath` with the `execPath` option', t => {
@@ -42,6 +46,7 @@
 		path.delimiter,
 	);
 	t.is(pathEnv[pathEnv.length - 2], path.resolve(process.cwd(), 'test'));
+	t.end();
 });
 
 test('the `execPath` option is relative to the `cwd` option', t => {
@@ -51,4 +56,5 @@
 		cwd: '/dir',
 	}).split(path.delimiter);
 	t.is(pathEnv[pathEnv.length - 2], path.normalize('/dir/test'));
+	t.end();
 });
